packages/status-js/src/client/chat.ts

```diff
@@ -17,12 +17,15 @@ import type { ImageMessage } from '~/src/proto/communities/v1/chat_message'
 import type { CommunityChat } from '~/src/proto/communities/v1/communities'
 import type { WakuMessage } from 'js-waku'
 
+type DeletedChatMessage = { deletedTimestamp: bigint }
```

Comment:
use `Date`

Reply:
Thinking about it again, the user-facing value are timestamps. A clock is responsible for the order of messages and can potentially be `timestamp + 1` – we don't want to show future dates in the UI.

Also, I would propose to name the fields `deletedAt`, `editedAt` for the public API. Consumers don't really need to know about implementational details such as timestamps or clocks.